CONTRIBUTING.md

```diff
@@ -82,7 +81,7 @@ Ungit has two major parts; the server and the UI. The server exposes a REST inte
 
 ### Running tests
 
-`grunt test` will run both unit tests, REST-interface tests, and click tests. `grunt unittest` only runs the tests in the test/ folder, `grunt clicktest` runs only the tests in the clicktests/ folder. Install mocha (`npm install -g mocha`) to run specific tests in the test folder and get better stack traces: `mocha test/spec.git-api.js`.
+`npm test` will run both unit tests, REST-interface tests, and click tests. `npm run unittest` only runs the tests in the test/ folder, `npm run clicktest` runs only the tests in the clicktests/ folder. Install mocha (`npm install -g mocha`) to run specific tests in the test folder and get better stack traces: `mocha test/spec.git-api.js`.
```

Comment:
```suggestion
`npm test` will run both unit tests, REST-interface tests, and click tests. `npm run unittest` only runs the tests in the `test/` folder, `npm run clicktest` runs only the tests in the `clicktests/` folder. Install Mocha (`npm install -g mocha`) to run specific tests in the test folder and get better stack traces: `mocha test/spec.git-api.js`.
```

Reply:
Instead of a global mocha, can we use
1. `npx mocha test/spec.git-api.js` or
2. `npm run unittest -- test/spec.git-api.js`

or something like that?

Reply:
The second option already works but needs `--no-config` so mocha does not not merge the spec files from the config file. But you would lose the the other settings like `timeout` and `file`:
`npm run unittest -- test/spec.git-api.js --no-config`